src/meson.build

```diff
@@ -104,10 +104,15 @@ criterion = declare_dependency(
 	dependencies: deps,
 	link_with: [libcriterion])
 
+pkgconfig_rpath = ''
+if get_option('rpath-in-pkgconfig').enabled() or (get_option('rpath-in-pkgconfig').auto() and  prefix.startswith('/home/'))
```

Comment:
micro-nit: extra space in `and  prefix.startswith('/home/')` after the `and`

Reply:
Fixed. Thanks for noticing.